tools/slo-generator/slo_generator/compute.py

```diff
@@ -41,36 +40,34 @@ def compute(slo_config,
         timestamp (int, optional): UNIX timestamp. Defaults to now.
         client (obj, optional): Existing metrics backend client.
         do_export (bool, optional): Enable / Disable export. Default: False.
-        backend_obj (obj) (optional): Backend object (if unset, will be
-            imported dynamically from slo_config). Use when you want to try new
-            backends that are not implemented in the backends/ folder.
-        backend_method (str) (optional): Backend method (if unset, will be
-            imported dynamically from slo_config). Use when you want to try new
-            backends that are not implemented in the backends/ folder.
-            This must return a tuple (n_good_events[int], n_bad_events[int]).
-            The method must take the following arguments:
-                - timestamp (int): query timestamp.
-                - window (int): query window duration.
-        backend_config (dict) (optional): Backend config.
```

Comment:
Remove feeding custom backend directly to `compute` method. Backend should be added to local fork in the corresponding `backends/` folder (see "Extending SLO Generator" section in README.md).